docs/markdown/30-testing/01-bats-setup.md

```diff
@@ -28,6 +28,4 @@ Notes:
 
 Easy way to use TDD method during development
 
-Notes:
-
 Speaker **Thibauld**
```

Comment:
put speaker line as 1st line of the notes block

Reply:
Done.